src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -0,0 +1,377 @@
+    setMode(mode === OPENING ? OPENED : CLOSED)
+  })
+  const handleOptIn = React.useCallback(() => {
+    onOptIn()
+    setTimeout(() => window.Beacon('open'), 100)
+  })
+  React.useEffect(() => {
+    if (optedIn && window.Beacon) {
+      window.Beacon('on', 'open', () => setMode(OPENED))
+      window.Beacon('on', 'close', () => setMode(CLOSED))
+    }
+  }, [optedIn, window.Beacon])
+
+  return (
+    <React.Fragment>
+      {!optedIn && (
+        <Transition
+          native
+          items={mode === OPENING || mode === OPENED}
+          from={{ opacity: 0, enterProgress: 0, blocking: false }}
+          enter={{ opacity: 1, enterProgress: 1, blocking: true }}
+          leave={{ opacity: 0, enterProgress: 0, blocking: false }}
+          onDestroyed={handleToggleEnd}
+          config={springs.smooth}
+        >
+          {show =>
+            show &&
+            /* eslint-disable react/prop-types */
+            (({ opacity, enterProgress, blocking }) => (
+              <OptInDialogue
+                style={{
+                  pointerEvents: blocking ? 'auto' : 'none',
+                  opacity,
+                  transform: enterProgress.interpolate(
+                    v => `
+                      translate3d(0, ${(1 - v) * 20}px, 0)
+                    `
+                  ),
+                }}
+                onClose={handleClose}
+                onOptIn={handleOptIn}
+              />
+            ))
+          /* eslint-enable react/prop-types */
+          }
+        </Transition>
+      )}
+      <ToggleDialogueButton
+        open={mode === OPENED || mode === OPENING}
+        onToggle={handleToggle}
+      />
+    </React.Fragment>
+  )
+})
+
+HelpOptIn.propTypes = {
+  onOptIn: PropTypes.func.isRequired,
+  optedIn: PropTypes.bool.isRequired,
+}
+
+const ToggleDialogueButton = ({ open, onToggle }) => {
+  return (
+    <RoundButton
+      onClick={onToggle}
+      css={`
+        margin-left: calc(100% - 60px);
+        margin-top: ${2 * GU}px;
+      `}
+    >
+      <Transition
+        native
+        items={open}
+        from={{ opacity: 0, enterProgress: 0 }}
+        enter={{ opacity: 1, enterProgress: 1 }}
+        leave={{ opacity: 0, enterProgress: 1 }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          /* eslint-disable react/prop-types */
+          (({ opacity, enterProgress }) => (
+            <AnimatedDiv
+              style={{
+                opacity,
+                transform: enterProgress.interpolate(
+                  v => `rotate(${45 * (1 - v)}deg)`
+                ),
+              }}
+            >
+              <IconClose
+                color={theme.gradientText}
+                css={`
+                  width: auto;
+                  height: 22px;
+
+                  & path {
+                    fill: ${theme.gradientText};
+                    transform: scale(2.2);
+                    opacity: 1;
```

Comment:
```suggestion
```

Reply:
The opacity is at `0.7` by default on this icon but the design's show a fully opaque one.

Reply:
Oooh ok, let’s leave it like that until we update the icons then!